plugins/outputs/bigquery/sample.conf

```diff
@@ -14,3 +14,7 @@
 
   ## Character to replace hyphens on Metric name
   # replace_hyphen_to = "_"
+
+  ## Write all metrics in a single compact table
+  # compact_table = false
+  # compact_table_name = ""
```

Comment:
How about using compact tables if the `compact_table_name` is not empty and remove the `compact_table`  flag?